examples/seal-key/seal-key.go

```diff
@@ -28,14 +29,39 @@ import (
 	"github.com/chrisccoulson/ubuntu-core-fde-utils"
 )
 
+type pathList []string
+
+func (l *pathList) String() string {
```

Comment:
pathList is defined as a slice which is essentially just a pointer to an array with len and cap. So it should be fine to avoid the "*" here and below, i.e. `func (l pathList) String() {...` (unless I miss something here?).

Reply:
Passing the receiver by value seems ok for String(), but Set() requires a pointer because it calls append(), so we need to update the receiver value for it to work.

Some documentation online suggests that you shouldn't mix pointer / value receivers (eg, https://tour.golang.org/methods/8) which is why I used a pointer receiver here, although I don't mind changing it.